features/registering_resources.feature

```diff
@@ -29,8 +29,3 @@ Feature: Registering resources
     When I put a new item into panopticon whose slug is already taken
     Then I should receive an HTTP 409 response
       And the relevant artefact should not be updated
-
-  Scenario: Deleting an item
```

Comment:
There are some corresponding step definitions in registration_steps.rb that could be removed